### DISCUSSION TITLE
Any one use plane with Podman instead Docker?

### DISCUSSION THREAD

**Discussion by calvin2021y** [jamsocket/plane#826] [Q&A]
Podman seems provide well rootless support .

**Comment by paulgb** [comment]
I haven't tried it, but there are two approaches.

The first is that, as I understand it, Podman supports a mostly-docker-compatible REST API over unix socket. If you point Plane at that unix socket, it _may_ just work, although some features like volumes may not.

The other approach is that Plane now supports "generic executors", it would be possible to write one that talks to `libpod` directly. Podman was actually one of the ideas that motivated the generic executors approach, although I likely won't have time to purse that myself.
